src/index.html

```diff
@@ -5,8 +5,393 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <a href="#home" class="header__logo-link">
+        <img src="images/logo.png" alt="logo icon">
+      </a>
+
+      <nav>
+        <ul class="header__nav-items-list">
+          <li class="header__nav-items">
+            <a href="#apple" class="header__nav-link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-items">
+            <a href="#samsung" class="header__nav-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-items">
+            <a href="#smatrphones" class="header__nav-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li data-qa="nav-hover" class="header__nav-items">
+            <a href="#laptops&Computers" class="header__nav-link">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-items">
+            <a href="#gadgets" class="header__nav-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-items">
+            <a href="#tablets" class="header__nav-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-items">
+            <a href="#photo" class="header__nav-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-items">
+            <a href="#video" class="header__nav-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="/images/imac.jpeg"
+            alt="mac icon"
```

Comment:
it's good to make alt attribute more descriptive
```suggestion
            alt="Picture of APPLE A1419 iMac 27" Retina 5K Monoblock"
```